extension/public/manifest.json

```diff
@@ -3,7 +3,19 @@
   "name": "WiseGPT",
   "description": "Autocomplete and personal prompts for chat GPT.",
   "version": "1.0",
-  "permissions": ["activeTab", "tabs", "storage", "scripting"],
+  "key": "MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAnS7bW0bTyhZKOA1W2yf3DFc2RwAlETDyf00EgdfeBwc1+qx1XkQdoiBklBwAfZK14hXMgD37mTjprm/5eB8gPT/sDoWWFZCdjlq+4HkPBTEw8n3frvA1nqYmWZ5LHUHkJrjGcghwSw64Mjsy52LG2eC9TLP7PH8dxAy9Los4p6HnpnV3wPJm7H4bHojeHXL/aX7xxecShnJHvkxRNWKw3Egm0Oq7eYC7b+E9A5xib9DVtSWYf7+Rdh2rqt6RYbo3PMPKV0NU/DyX7uUfhrKFDbas+y3qQCj8VyqXoqkmuEdw/c0jtBwuUhleyt/TizaJxo735IAjwu3u+U//5qGRtQIDAQAB",
+  "permissions": [
+    "identity",
+    "activeTab",
+    "tabs",
+    "storage",
+    "scripting",
+    "webRequest"
+  ],
+  "oauth2": {
+    "client_id": "986913113073-5j906si3om6op54prl17mpg94gp4c8g8.apps.googleusercontent.com",
+    "scopes": ["openid", "email", "profile"]
+  },
```

Comment:
This refers to the client_id that we set up with google. Scopes are what information we allow google to share with Wise.